specification/postgresqlhsc/resource-manager/Microsoft.DBforPostgreSQL/stable/2022-11-08/examples/ClusterCreate.json

```diff
@@ -0,0 +1,166 @@
+{
+  "parameters": {
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "TestGroup",
+    "clusterName": "hsctestsg",
+    "api-version": "2022-11-08",
+    "parameters": {
+      "location": "westus",
+      "properties": {
+        "administratorLoginPassword": "p@ssword1234",
```

Comment:
>p@ssword1234 [](http://example.com/codeflow?start=39&length=12)

CredScan Test is failing due to the sample passwords mentioned in examples. I would recommend to remove these sample passwords in example, and keep these properties empty in examples.

Reply:
Fixed passwords